ext/hal/ti/simplelink/Makefile

```diff
@@ -0,0 +1,6 @@
+ifdef CONFIG_HAS_CC3220SDK
+ZEPHYRINCLUDE +=-I$(srctree)/ext/hal/ti/simplelink/source/ti/devices/cc32xx
+ZEPHYRINCLUDE +=-I$(srctree)/ext/hal/ti/simplelink/source/ti/devices/cc32xx/inc
+ZEPHYRINCLUDE +=-I$(srctree)/ext/hal/ti/simplelink/source/ti/devices/cc32xx/driverlib
+KBUILD_CFLAGS += -DUSE_CC3200_ROM_DRV_API
```

Comment:
Yes, the cc3200sdk will be deprecated with the cc3200 board.
However, I just noticed something about KBUILD_CFLAGS.  This patch still works with the CC3220, but to be more correct, it should be KBUILD_CFLAGS += -DUSE_CC3220_ROM_DRV_API.